main/ansible/APKBUILD

```diff
@@ -38,9 +38,9 @@ package() {
 
 	mkdir -p "$pkgdir"/usr/share/doc/$pkgname/examples/
 	install -m644 examples/* \
-		"$pkgdir"/usr/share/doc/$pkgname/examples/
+		"$pkgdir"/usr/share/doc/$pkgname/examples/ || true
```

Comment:
that looks strange

Reply:
this is strange...
`install` command ignore subdirectories and the return non zero code.
This make the build fail.

Actually, we don't realy need what's inside `examples` but since it was there before, I didn't want to remove it.
So what should we do ?